src/test/java/org/opensearch/security/support/Base64HelperTest.java

```diff
@@ -41,67 +42,121 @@ private static final class NotSafeSerializable implements Serializable {
         SourceFieldsContext sourceFieldsContext = new SourceFieldsContext(new SearchRequest(""));
-        Assert.assertEquals(sourceFieldsContext.toString(), ds(sourceFieldsContext).toString());
+        Assert.assertEquals(sourceFieldsContext.toString(), ds(sourceFieldsContext, PackageBehavior.NONE).toString());
     }
 
     @Test
     public void testHashMap() {
         HashMap map = new HashMap();
-        Assert.assertEquals(map, ds(map));
+        Assert.assertEquals(map, ds(map, PackageBehavior.NONE));
     }
 
     @Test
     public void testArrayList() {
         ArrayList list = new ArrayList();
-        Assert.assertEquals(list, ds(list));
+        Assert.assertEquals(list, ds(list, PackageBehavior.NONE));
+    }
+
+    @Test
+    public void testStringWithRewriteOdfePackage() {
+        String string = "string";
+        Assert.assertEquals(string, ds(string, PackageBehavior.REWRITE_AS_ODFE));
+    }
+
+    @Test
+    public void testIntegerWithRewriteOdfePackage() {
+        Integer integer = Integer.valueOf(0);
+        Assert.assertEquals(integer, ds(integer, PackageBehavior.REWRITE_AS_ODFE));
+    }
+
+    @Test
+    public void testDoubleWithRewriteOdfePackage() {
+        Double number = Double.valueOf(0.);
+        Assert.assertEquals(number, ds(number, PackageBehavior.REWRITE_AS_ODFE));
+    }
+
+    @Test
+    public void testInetSocketAddressWithRewriteOdfePackage() {
+        InetSocketAddress inetSocketAddress = new InetSocketAddress(0);
+        Assert.assertEquals(inetSocketAddress, ds(inetSocketAddress, PackageBehavior.REWRITE_AS_ODFE));
+    }
+
+    @Test
+    public void testPatternWithRewriteOdfePackage() {
+        Pattern pattern = Pattern.compile(".*");
+        Assert.assertEquals(pattern.pattern(), ((Pattern) ds(pattern, PackageBehavior.REWRITE_AS_ODFE)).pattern());
+    }
+
+    @Test
+    public void testUserWithRewriteOdfePackage() {
+        User user = new User("user");
+        Assert.assertEquals(user, ds(user, PackageBehavior.REWRITE_AS_ODFE));
+    }
+
+    @Test
+    public void testSourceFieldsContextWithRewriteOdfePackage() {
+        SourceFieldsContext sourceFieldsContext = new SourceFieldsContext(new SearchRequest(""));
+        Assert.assertEquals(sourceFieldsContext.toString(), ds(sourceFieldsContext, PackageBehavior.REWRITE_AS_ODFE).toString());
+    }
+
+    @Test
+    public void testHashMapWithRewriteOdfePackage() {
+        HashMap map = new HashMap();
+        Assert.assertEquals(map, ds(map, PackageBehavior.REWRITE_AS_ODFE));
+    }
+
+    @Test
+    public void testArrayListWithRewriteOdfePackage() {
+        ArrayList list = new ArrayList();
+        Assert.assertEquals(list, ds(list, PackageBehavior.REWRITE_AS_ODFE));
     }
 
     @Test(expected = OpenSearchException.class)
     public void notSafeSerializable() {
-        serializeObject(new NotSafeSerializable());
+        serializeObject(new NotSafeSerializable(), PackageBehavior.NONE);
```

Comment:
Apologies if I don't understand, but why not duplicate this test, same as above?

Reply:
Missed it, thanks for catching!